docs/api2.yaml

```diff
@@ -283,9 +283,391 @@ paths:
                       crowdactionID: already participating
       security:
         - FirebaseAuthorizer: []
-  '/profiles/{userID}':
+  "/crowdactions/{crowdactionID}/comments":
+    get:
+      tags:
+        - Crowdaction
+      summary: Get comments a specific crowdaction
+      parameters:
+        - $ref: "#/components/parameters/ApiVersionParameter"
+        - name: crowdactionID
+          in: path
+          required: true
+          schema:
+            type: string
+        - in: query
+          name: password
+          description: Only include if crowdaction requires password
+          required: false
+          schema:
+            type: string
+            format: password
+      responses:
+        "200":
+          description: Comments
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+                    default: success
+                  data:
+                    type: array
+                    items:
+                      $ref: "#/components/schemas/Comment"
+        "401":
+          description: Unauthorized (Invalid password)
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+                    default: fail
+                  data:
+                    type: string
+                    example:
+                      password: Invalid or missing password
+        "403":
+          $ref: "#/components/responses/UnsupportedClientVersion"
+        "404":
+          description: Crowdaction not found
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+                    default: fail
+                  data:
+                    type: string
+                    example:
+                      crowdactionID: crowdaction does not exist
+    post:
+      tags:
+        - "Crowdaction"
+      summary: Create a new comment on a crowdaction
```

Comment:
We need an optional parameter to include a parent comment, comments can be nested x1.
```
|- 1
|- 2
|- 3
|   |- 4
|   |- 6
|
|- 5
```
Probably a bad example, but I'm sure you get the idea. Maybe we can discuss this?

Reply:
So we DO want multi level comments?  
I'll look into that. I think I need to refactor the endpoint a bit.